src/payment/strategies/amazon-maxo/create-amazon-maxo-payment-processor.ts

```diff
@@ -0,0 +1,22 @@
+import { createRequestSender } from '@bigcommerce/request-sender';
+import { getScriptLoader } from '@bigcommerce/script-loader';
+
+import { CheckoutStore } from '../../../checkout';
+import PaymentMethodActionCreator from '../../payment-method-action-creator';
+import PaymentMethodRequestSender from '../../payment-method-request-sender';
+
+import AmazonMaxoPaymentProcessor from './amazon-maxo-payment-processor';
+import AmazonMaxoScriptLoader from './amazon-maxo-script-loader';
+
+export default function createAmazonMaxoPaymentProcessor(store: CheckoutStore): AmazonMaxoPaymentProcessor {
```

Comment:
would be good if you can add a unit test for this

Reply:
Test was added on amazon-maxo-payment-processor.spec.ts.